backend/src/main/java/com/example/demo/stock/ApiController.java

```diff
@@ -9,12 +9,20 @@
 @Data
```

Comment:
```suggestion
@RequiredArgsConstructor
```

Overkill vor controller, only need the constructor